docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -1,53 +1,180 @@
-.. _2017_risk_exposure_template:
+.. _2017_exposure_template:
 
-----------------------
+======================
 Risk Exposure Template
-----------------------
+======================

```

Comment:
If this title is at the same section level as the "Full Name of Risk Exposure in GBD 2017" section below (i.e. they are both decorated with the same `======` underline/overline), then the Risk Exposure Models index page (`/vivarium_research/docs/build/html/gbd2017_models/risks/index.html`) shows "Full Name of Risk Exposure in GBD 2017" in the table of contents.  Build the HTML to see what I mean.  That's why in the cause template I used a different, unique heading for "Cause Model Template," namely an overline/underline of `--------`.